<?xml version="1.0" ?>
<launch>
    <param name="measurementServerName" value="/point_cloud_server"/>
    <node pkg="robot_arm_3d_scan" type="point_cloud_server" name="point_cloud_server" output="screen"/>
</launch>


